include/robots/tank.h

```diff
@@ -70,6 +70,10 @@ class Tank
 
     void stopReadingFromNetwork();
 
+    float getLeft() const;
```

Comment:
Haha I wonder how many branches this change has been made in independently

Reply:
I'm sure we'll get some lovely merge conflicts when all those branches eventually come back together!